assets/css/alert-banner.scss

```diff
@@ -4,6 +4,9 @@
   background-color: #095292;
   align-items: center;
   padding: 16px 221px 16px 50px;
+  position: sticky;
+  top: 0;
+  z-index: 2;
```

Comment:
Unless there's something else that might need to cover up this banner (which I doubt since it's sticky), you don't need to be shy about setting the z-index to something big, like `99`, `999`, something like that. It will ensure that nothing appears on top of the banner, and we can always adjust the number later if needed.

Reply:
Ahh, good idea.